cli/index.ts

```diff
@@ -126,21 +131,50 @@ createYargsCli({
         const result = await init(
           argv["project-dir"],
           {
-            warehouse: argv["warehouse"],
+            warehouse: argv.warehouse,
             gcloudProjectId: argv["gcloud-project-id"]
           },
           argv["skip-install"]
         );
 
-        console.log(commandOutputFmtString, "Directories created:");
-        result.dirsCreated.forEach(dir => console.log(dir));
-        console.log(commandOutputFmtString, "Files written:");
-        result.filesWritten.forEach(file => console.log(file));
+        writeStdOut(commandOutput("Directories created:"));
+        result.dirsCreated.forEach(dir => writeStdOut(dir));
+        writeStdOut(commandOutput("Files written:"));
+        result.filesWritten.forEach(file => writeStdOut(file));
         if (result.installedNpmPackages) {
-          console.log(commandOutputFmtString, "NPM packages successfully installed.");
+          writeStdOut(commandOutput("NPM packages successfully installed."));
         }
       }
     },
+    {
+      format: "init-creds <warehouse> [project-dir]",
+      description:
+        "Creates a .df-credentials.json file for dataform to use when accessing your warehouse.",
+      positionalOptions: [warehouseOption, projectDirMustExistOption],
+      options: [],
+      processFn: argv => {
+        const credentials = () => {
+          switch (argv.warehouse) {
+            case "bigquery": {
+              return getBigQueryCredentials();
+            }
+            case "redshift": {
```

Comment:
The redshift connector works with postgres as well. Since some users may want to use PG as well, would be great to add that option as well.

Reply:
So generally I agree, though right now the dataform package doesn't really support (in the true meaning of the word) postgres - we should really add a WarehouseType entry, register the redshift dbadapter to support that case, etc. Would prefer to do that separately, as I'm not sure what the knock-on effects might be.

Reply:
agreed